<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColPropForm</class>
 <widget class="QWidget" name="ColPropForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>329</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QPlainTextEdit" name="te_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_descr">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>261</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширина</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_width">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Точность</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_precision">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Формула</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_expression">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
